packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 49.2.0 - July 24, 2020
+
+## Fixed
+
+#### docs
+
+- Update [Remote Development - Containers](https://developer.salesforce.com/tools/vscode/en/user-guide/remote-development/) ([PR #2352](https://github.com/forcedotcom/salesforcedx-vscode/pull/2352))
+
+#### salesforcedx-vscode-core
+
+- We fixed some minor under-the-hood bugs.
+
+# 49.1.0 - July 18, 2020
```

Comment:
The changes for 49.1.0 were done directly in master and I forgot to port them back to develop.